src/flood_forecaster/utils/database_helper.py

```diff
@@ -0,0 +1,127 @@
+import configparser
+import os
+from sqlalchemy import create_engine, inspect, text
+from sqlalchemy.engine import URL
+from sqlalchemy.exc import SQLAlchemyError
+
+
+class DatabaseConnection:
+    def __init__(self, config_file_path: str) -> None:
+        """
+        Initialize the database connection using parameters from a config file.
+
+        :param config_file_path: Path to the configuration file
+        """
+        self.config = self._load_config(config_file_path)
+        self.dbname = self.config.get("database", "dbname")
+        self.user = self.config.get("database", "user")
+        self.host = self.config.get("database", "host")
+        self.port = self.config.get("database", "port")
+        self.password = os.environ.get("POSTGRES_PASSWORD")
+
+        try:
+            url = URL.create(
+                drivername="postgresql",
+                username=self.user,
+                password=self.password,
+                host=self.host,
+                port=self.port,
+                database=self.dbname
+            )
+            self.engine = create_engine(url)
+            print(f"Connected to database '{self.dbname}'")
+        except SQLAlchemyError as e:
+            print(f"Failed to connect to database: {str(e)}")
+            raise
+
+    @staticmethod
+    def _load_config(config_file_path: str) -> configparser.ConfigParser:
```

Comment:
First version already in `configuration.py`. Unify? Move there?

Reply:
Sorry, missed that first, then I thought is better to keep all db related stuff together and keep configuration for API config - as might have many of them.